server/src/main/java/fr/uga/l3miage/tp1/exo2/models/NetflixAccoutEntity.java

```diff
@@ -0,0 +1,20 @@
+package fr.uga.l3miage.tp1.exo2.models;
+
+import javax.persistence.*;
+import java.math.BigInteger;
+import java.time.LocalDate;
+
+@Entity
+@Table(name = "netflix_account")
+public class NetflixAccoutEntity {
+    @Id
+    @Column(name = "id")
+    private BigInteger id ;
+    @Column(name = "nb_devices")
+    private int nombreDevices ;
+    @Column(name = "typeAccount")
+    private String typeAccount ;
+    @Column(name = "uuid_user")
+    private String uuid_user ;
+
```

Comment:
Ici il te manque la relation retour
```suggestion
@OneToOne
@JoinColumn(name="uuid_user",  referencedColumnName="uuid")
private final NetflixUserEntity user;
```

avec l'annotation `@JoinColumn(name="uuid_user",  referencedColumnName="uuid")` qui permet de te montrer comme faire la jointure entre les tables